.ci/macos-steps.yaml

```diff
@@ -12,9 +12,9 @@ steps:
 # Install Build Dependencies
 - script: |
     set -e
-    sudo xcode-select --switch /Applications/Xcode_10.1.app/Contents/Developer
     unset BOOST_ROOT
     pip install cython numpy pandas zipp configparser
+    brew install libomp
```

Comment:
Wondering if `export OMP_NUM_THREADS=1` would 'solve' the issue we see on the tests.

Reply:
@zoq tests still failed. I'm going to debug this further in pinpoint the reason why they fail

Reply:
Thanks!